pontoon/translate/static/js/translate/TranslateView.es6

```diff
@@ -0,0 +1,157 @@
+import {Entity, Project} from './models.js';
+import TranslationEditor from './TranslationEditor.js';
+import TranslationSelector from './TranslationSelector.js';
+
+let CSSTransitionGroup = React.addons.CSSTransitionGroup;
+
+
+export default class TranslateView extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loaded: false,
+      entities: [],
+      projects: [],
+    };
+  }
+
+  componentDidMount() {
+    $.when(
+      Entity.fetchAll(this.props.currentProject.slug, this.props.locale.code),
+      Project.fetchAll()
+    ).then((entities, projects) => {
```

Comment:
However, it seems like you aren't using es6-standard promises, and are using jquery promises. Best to be consistent then.